base/java-tools/templates/pretty_print_cert_command_wrapper.in

```diff
@@ -87,11 +87,7 @@ JAVA_OPTIONS=""
 ##      order this command wrapper uses to find jar files.                   ##
 ###############################################################################
 
-CP=${JNI_JAR_DIR}/jss4.jar
-CP=/usr/share/java/commons-codec.jar:${CP}
-CP=/usr/share/java/ldapjdk.jar:${CP}
-CP=/usr/share/java/${PRODUCT}/pki-cmsutil.jar:${CP}
-CP=/usr/share/java/${PRODUCT}/pki-tools.jar:${CP}
+CP="${PKI_LIB}/*"
```

Comment:
@edewata Will this path get populated on tools-only (no server) systems? It looks like it's in `/usr` and so should be provided by the RPM package (vs. `pkispawn` or something) -- and the server I tested it on didn't have an instance. I'm inclined to think at worst the wrong RPM package will provide the location but I think it should be fine.

Reply:
If I understand your question correctly, the `PKI_LIB` environment variable is set by `pki.conf`, which is a common configuration file for client and server (so it should be available in tools-only systems). The default file is defined in `/usr/share/pki/etc/pki.conf` (which comes with pki-base package), but it can be overridden with a custom system-wide config at '/etc/pki/pki.conf' or a user-specific config at `~/.dogtag/pki.conf`. The path is now hard-coded in `base/common/share/etc/pki.conf`, but if we want we can use the value provided by RPM.

Reply:
Sorry, I was wondering what RPM package owned the path. 

`pki-base-10.8.3-1.fc32.noarch <- pki-base-java-10.8.3-1.fc32.noarch <- pki-tools-0:10.8.3-1.fc32.x86_64`, so we're good.